examples/platforms/beagleboneblack/default

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
 #
-#  Copyright (c) 2017, The OpenThread Authors.
+#  Copyright (c) 2021, The OpenThread Authors.
```

Comment:
@jwhui Should this be the year when this file is created?

Reply:
We should retain the original year, but can indicate the update as follows:
```suggestion
#  Copyright (c) 2017-2021, The OpenThread Authors.
```